benchopt/plotting/generate_html.py

```diff
@@ -0,0 +1,296 @@
+import shutil
+import glob
+import webbrowser
+import click
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from mako.template import Template
+from pathlib import Path
+from datetime import datetime
+
+from ..constants import PLOT_KINDS
+from .plot_histogram import plot_histogram  # noqa: F401
+from .plot_objective_curve import plot_objective_curve  # noqa: F401
+from .plot_objective_curve import plot_suboptimality_curve  # noqa: F401
+from .plot_objective_curve import plot_relative_suboptimality_curve  # noqa: F401 E501
+
+
+ROOT = Path(__file__).parent / "html"
+BUILD_DIR = Path("build")
```

Comment:
Instead of `build`, maybe better to call this `html`
```suggestion
HTML_DIR = Path("html")
```